src/spyglass/spikesorting/v0/spikesorting_recording.py

```diff
@@ -395,21 +395,20 @@ def make(self, key):
             folder=recording_path, chunk_duration="10000ms", n_jobs=8
         )
 
-        IntervalList.insert1(
-            {
+        interval_key = IntervalList().cautious_insert1(
+            key={
                 "nwb_file_name": key["nwb_file_name"],
                 "interval_list_name": recording_name,
                 "valid_times": sort_interval_valid_times,
                 "pipeline": "spikesorting_recording_v0",
             },
-            replace=True,
-        )
+            approx_name=True,
+        )  # removed replace=True 2024-04-22
```

Comment:
Could make redoing this step tricky for users. They would have to know they need to manually delete the interval list when deleting here